src/App.js

```diff
@@ -674,7 +674,7 @@ export default function App() {
   const footer = (
   <div className={classes.footer}>
     <Typography variant="body1">
-      Developed by ToniqLabs &copy; All rights reserved 2021<br /><a href="https://docs.google.com/document/d/13aj8of_UXdByGoFdMEbbIyltXMn0TXHiUie2jO-qnNk/edit" target="_blank">Terms of Service</a>
+      Developed by ToniqLabs &copy; All rights reserved 2021<br /><a href="https://docs.google.com/document/d/13aj8of_UXdByGoFdMEbbIyltXMn0TXHiUie2jO-qnNk/edit" target="_blank" rel="noreferrer">Terms of Service</a>
```

Comment:
Let's add `noopener` to make it in total `noopener noreferrer`. Pretty cool that the build was catching this!

- [ ] Add `noopener`